### PR TITLE
[th2-2689] adaptation to book/page cradleapi

### PR DESCRIPTION
**Pull request by andreydrobynin**

### CODE REVIEW

**Comment by andreydrobynin** [comment]
Related to https://github.com/th2-net/th2-common-j/pull/145, https://github.com/th2-net/th2-store-common/pull/12

File: src/main/java/com/exactpro/th2/mstore/AbstractMessageStore.java [hunk 1]
```diff
@@ -173,68 +174,68 @@ public final void handle(T messageBatch) {
...
         });
     }
 
-    private static String formatStoredMessageBatch(StoredMessageBatch storedMessageBatch, boolean full) {
-        ToStringBuilder builder = new ToStringBuilder(storedMessageBatch, NO_CLASS_NAME_STYLE)
-                .append("stream", storedMessageBatch.getStreamName())
-                .append("direction", storedMessageBatch.getId().getDirection())
-                .append("batch id", storedMessageBatch.getId().getIndex());
+    private static String formatMessageBatchToStore(MessageBatchToStore messageBatchToStore, boolean full) {
+        ToStringBuilder builder = new ToStringBuilder(messageBatchToStore, NO_CLASS_NAME_STYLE)
```
Comment (Nikita-Smirnov-Exactpro) [inline src/main/java/com/exactpro/th2/mstore/AbstractMessageStore.java:229]: Please add book name to both option

File: src/main/java/com/exactpro/th2/mstore/AbstractMessageStore.java [hunk 2]
```diff
@@ -55,7 +56,7 @@
     private final ScheduledExecutorService drainExecutor = Executors.newSingleThreadScheduledExecutor();
     private final Map<SessionKey, SessionData> sessionToHolder = new ConcurrentHashMap<>();
     private final MessageStoreConfiguration configuration;
-    private final Map<CompletableFuture<Void>, StoredMessageBatch> asyncStoreFutures = new ConcurrentHashMap<>();
+    private final Map<CompletableFuture<Void>, MessageBatchToStore> asyncStoreFutures = new ConcurrentHashMap<>();
     private volatile ScheduledFuture<?> future;
 
     public AbstractMessageStore(
```
Comment (Nikita-Smirnov-Exactpro) [inline src/main/java/com/exactpro/th2/mstore/AbstractMessageStore.java:62]: Please add book name into all logging where session_alias or direction or sequence are written

File: src/test/java/com/exactpro/th2/mstore/TestCaseMessageStore.java [hunk 3]
```diff
@@ -143,9 +142,9 @@ private T deliveryOf(M... messages) {
         return createDelivery(List.of(messages));
     }
 
-    private static void assertStoredMessageBatch(StoredMessageBatch batch, String streamName, Direction direction, int count) {
+    private static void assertMessageBatchToStore(MessageBatchToStore batch, String sessionAlias, Direction direction, int count) {
         assertEquals(ProtoUtil.toCradleDirection(direction), batch.getDirection());
-        assertEquals(streamName, batch.getStreamName());
+        assertEquals(sessionAlias, batch.getSessionAlias());
```
Comment (Nikita-Smirnov-Exactpro) [inline src/test/java/com/exactpro/th2/mstore/TestCaseMessageStore.java:147]: Let's add book name check

**Review by Nikita-Smirnov-Exactpro — CHANGES_REQUESTED** [review]

**Comment by Nikita-Smirnov-Exactpro** [comment]
Use a branch related to book page as target branch

File: src/main/java/com/exactpro/th2/mstore/AbstractMessageStore.java (hunk 1 above)
Comment (andreydrobynin) [reply src/main/java/com/exactpro/th2/mstore/AbstractMessageStore.java:256]: done

File: src/main/java/com/exactpro/th2/mstore/AbstractMessageStore.java (hunk 2 above)
Comment (andreydrobynin) [reply src/main/java/com/exactpro/th2/mstore/AbstractMessageStore.java]: done

File: src/test/java/com/exactpro/th2/mstore/TestCaseMessageStore.java (hunk 3 above)
Comment (andreydrobynin) [reply src/test/java/com/exactpro/th2/mstore/TestCaseMessageStore.java:149]: done

File: src/main/java/com/exactpro/th2/mstore/AbstractMessageStore.java [hunk 4]
```diff
@@ -336,7 +337,7 @@ private void drainHolder(SessionKey key, SessionBatchHolder holder, boolean forc
             batch = holder.reset();
         }
         if (batch.isEmpty()) {
-            logger.debug("Holder for stream: '{}', direction: '{}' has been concurrently reset. Skip storing by scheduler",
+            logger.debug("Holder for session alias: '{}', direction: '{}' has been concurrently reset. Skip storing by scheduler",
```
Comment (Nikita-Smirnov-Exactpro) [inline src/main/java/com/exactpro/th2/mstore/AbstractMessageStore.java:340]: Must we add the `book name` property to the SessionKey class?

Comment (Nikita-Smirnov-Exactpro) [reply src/main/java/com/exactpro/th2/mstore/AbstractMessageStore.java:340]: This is a common idea and doesn't related to code line

Comment (andreydrobynin) [reply src/main/java/com/exactpro/th2/mstore/AbstractMessageStore.java]: done

**Review by Nikita-Smirnov-Exactpro — APPROVED** [review]

**Pull request merged**